tools/delete_imaging_upload.pl

```diff
@@ -0,0 +1,1712 @@
+#!/usr/bin/perl
+
+=pod
+
+=head1 NAME
+
+delete_mri_upload.pl -- Delete everything that was produced (or part of what was produced) by the imaging pipeline for a given set of imaging uploads
+                        IDs, all associated to the same C<tarchive>.
+                        
+=head1 SYNOPSIS
+
+perl delete_mri_upload.pl [-profile file] [-ignore] [-backup basename] [-protocol] [-form] [-uploadID list_of_uploadIDs]
+            [-type list_of_scan_types] [-defaced] [-nosqlbk] [-nofilesbk]
+
+Available options are:
+
+-profile           : name of the config file in C<../dicom-archive/.loris_mri> (defaults to C<prod>).
+
+-ignore            : ignore files whose paths exist in the database but do not exist on the file system.
+                     Default is to abort if such a file is found, irrespective of whether a backup file will
+                     be created by the script (see C<-nofilesbk> and C<-nosqlbk>). If this option is used, a 
+                     warning is issued and program execution continues.
+               
+-nofilesbk         : when creating the backup file for the deleted upload(s), do not backup the files produced by
+                     the imaging pipeline (default is to backup these files).
+               
+-backup <basename> : specify the base name of the backup file, which by default contains a copy of everything that the
```

Comment:
maybe relabel the option to backup_name?

When I read just `-backup`, I automatically think this is a boolean allowing for backup. I know the description explains what it is but maybe for clarity we could rename the option to backup_name or something similar?

Reply:
Ok for `backup_name`.